confmap/internal/e2e/expand_test.go

```diff
@@ -5,90 +5,99 @@ package e2etest
 
 import (
 	"context"
-	"fmt"
 	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"go.opentelemetry.io/collector/confmap"
-	"go.opentelemetry.io/collector/confmap/converter/expandconverter"
 	"go.opentelemetry.io/collector/confmap/provider/envprovider"
 	"go.opentelemetry.io/collector/confmap/provider/fileprovider"
-	"go.opentelemetry.io/collector/confmap/provider/yamlprovider"
 )
 
 // Test_EscapedEnvVars tests that the resolver supports escaped env vars working together with expand converter.
-func Test_EscapedEnvVars(t *testing.T) {
-	tests := []struct {
-		name   string
-		scheme string
-	}{
-		{
-			name:   "no_default_scheme",
-			scheme: "",
-		},
-		{
-			name:   "env",
-			scheme: "env",
-		},
-	}
-
+func Test_EscapedEnvVars_NoDefaultScheme(t *testing.T) {
 	const expandedValue = "some expanded value"
 	t.Setenv("ENV_VALUE", expandedValue)
+	t.Setenv("ENV_LIST", "['$$ESCAPE_ME','$$ESCAPE_ME']")
```

Comment:
Given that $$ENV isn't supported anymore, it's better to add `$${}` here I think?
```suggestion
	t.Setenv("ENV_LIST", "['$$ESCAPE_ME','$${ESCAPE_ME}','${ENV_VALUE}']")
```

Reply:
Good idea, added